tftest.py

```diff
@@ -49,7 +50,8 @@ class TerraformTestError(Exception):
   pass
 
 
-def parse_args(init_vars=None, tf_vars=None, targets=None, **kw):
+# def parse_args(init_vars=None, tf_vars=None, targets=None, **kw):
```

Comment:
There's no need to keep the old code in comments. ;)

Reply:
After going through part of the code, I have to say that while I like generalizing key/values here, and I understand the need to account for both Terraform and Terragrunt args, I would still like to preserve individual function arguments for their documentation aspect. Using this class is hard enough without having to devise arg names for functions.

What do you think of adding two sets of wrappers for the public methods, one with a `tf_` prefix, and one with a `tg_` prefix, with explicitly named args? The changes would be minimal, the old methods would still work, and users would be able to check with args can be passed where.

The alternative of course is to properly document all methods for both Terraform and Terragrunt.

What do you think?

Reply:
I agree that the documentation aspect of the defined args is preferable. My main concern was the maintainability of the function args as future releases of Terragrunt and Terraform incorporate new CLI args. 

In the case we do define the args for the public methods, should we include args from a specific range or include all args including deprecated ones?

Reply:
Absolutely, I think generalizing args makes complete sense. Shall I take it that the `tf_xxx` / `tg_xxx` approach is ok for you? We can either add them here, or I can do a quick PR right after this is merged. As for which args, I would just stay compatible with the current version, knowing adding new args will be super easy when there's a need.

Reply:
`tf_xxx`/ `tg_xxx` sounds good.  I'll add them to this PR.